controllers/schedule_it_utils_test.go

```diff
@@ -0,0 +1,49 @@
+// +build integration
+
+package controllers_test
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	k8upv1alpha1 "github.com/vshn/k8up/api/v1alpha1"
+	"github.com/vshn/k8up/cfg"
+)
+
+func (ts *ScheduleControllerTestSuite) givenScheduleResource() {
+	cfg.Config.OperatorNamespace = ts.NS
+	givenSchedule := ts.newScheduleSpec("test")
+	ts.EnsureResources(givenSchedule)
+	ts.givenSchedule = givenSchedule
+}
+
+func (ts *ScheduleControllerTestSuite) newScheduleSpec(name string) *k8upv1alpha1.Schedule {
+	return &k8upv1alpha1.Schedule{
+		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: ts.NS},
+		Spec: k8upv1alpha1.ScheduleSpec{
+			Backup: &k8upv1alpha1.BackupSchedule{
+				ScheduleCommon: &k8upv1alpha1.ScheduleCommon{
+					Schedule: "@daily-random",
+				},
+			},
+		},
+	}
+}
+
+func (ts *ScheduleControllerTestSuite) thenAssertEffectiveScheduleExists(expectedScheduleName string) {
+	list := ts.whenListEffectiveSchedules()
+	ts.Require().NotEmpty(list)
+	spec := list[0].Spec
+	ts.Require().NotEmpty(spec.EffectiveSchedules)
+	ref := spec.EffectiveSchedules[0]
+	ts.Assert().Equal(expectedScheduleName, ref.Name)
```

Comment:
Is there an advantage over this:

```suggestion
	assert.Equal(ts.T(), expectedScheduleName, ref.Name)
```

Reply:
it's exactly the same. just kind of a shortcut.